modules/io/python/drivers/io/kube_ssh.sh

```diff
@@ -5,16 +5,28 @@ shift
 
 if [[ "$POD_NAME" =~ ":" ]]; then
     KUBE_NS=${POD_NAME%%:*}
-    KUBE_POD_NAME=${POD_NAME#*:}
-
-    kubectl -n ${KUBE_NS} exec ${KUBE_POD_NAME} -c engine -- /bin/bash -lc "
-        cat /etc/hosts > /dev/null || true && \
-        source /etc/profile || true && \
-        source /etc/bash.bashrc || true && \
-        source ~/.profile || true && \
-        source ~/.bashrc || true && \
-        shopt -s huponexit 2>/dev/null || true && \
-        $*"
+    KUBE_POD_CONTAINER_NAME=${POD_NAME#*:}
+    if [[ "$KUBE_POD_CONTAINER_NAME" =~ ":" ]]; then
+        KUBE_POD_NAME=${KUBE_POD_CONTAINER_NAME%%:*}
+        KUBE_CONTAINER_NAME=${KUBE_POD_CONTAINER_NAME#*:}
+        kubectl -n ${KUBE_NS} exec ${KUBE_POD_NAME} -c ${KUBE_CONTAINER_NAME} -- /bin/bash -lc "
+            cat /etc/hosts > /dev/null || true && \
+            source /etc/profile || true && \
+            source /etc/bash.bashrc || true && \
+            source ~/.profile || true && \
+            source ~/.bashrc || true && \
+            shopt -s huponexit 2>/dev/null || true && \
+            $*"
+    else
+        kubectl -n ${KUBE_NS} exec ${KUBE_POD_CONTAINER_NAME} -c engine -- /bin/bash -lc "
```

Comment:
Remove the default `engine` here.

We could just rely on the `default-container` setting in Pod's YAML.

Reply:
Done.